crates/sui-json-rpc-types/src/sui_object.rs

```diff
@@ -876,3 +876,12 @@ impl From<ObjectInfo> for SuiObjectInfo {
         }
     }
 }
+
+#[derive(Debug, Deserialize, Serialize, JsonSchema, Clone, Eq, PartialEq)]
+#[serde(rename = "GetPastObjectRequest", rename_all = "camelCase")]
+pub struct SuiGetPastObjectRequest {
+    /// the ID of the queried object
+    pub object_id: ObjectID,
+    /// the version of the queried object. If None, default to the latest known version
+    pub version: SequenceNumber,
```

Comment:
```suggestion
    /// the version of the queried object.
    pub version: SequenceNumber,
```
This part seems to be no longer relevant. Could you also remove this in other places as well?